src/vivarium/framework/results/stratification.py

```diff
@@ -14,14 +16,15 @@ class Stratification:
     Each Stratification represents a set of mutually exclusive and collectively
     exhaustive categories into which simulants can be assigned.
 
-    The `Stratification` class has five fields: `name`, `sources`, `mapper`,
-    `categories`, and `is_vectorized`. The `name` is the name of the column
-    created by the mapper. The `sources` is a list of columns in the extended
-    state table that are the inputs to the mapper function.  Simulants will
-    later be grouped by this column (or these columns) during stratification.
-    `categories` is a set of values that the mapper is allowed to output. The
-    `mapper` is the method that transforms the source to the name column.
-    The method produces an output column by calling the mapper on the source
+    The `Stratification` class has six fields: `name`, `sources`, `mapper`,
```

Comment:
I guess this PR only adds to what existed already, but if we're going to go through each attribute one by one, we should probably use a systematic docstring format i.e. what we use for say, Builder 
https://github.com/ihmeuw/vivarium/blob/main/src/vivarium/framework/engine.py
I'd say "X class has # fields" is also unnecessary.

Reply:
Agree, and I'll do all that doc stuff in the final PRs for this epic. For now I only modified docs if they were categorically incorrect